### PR TITLE
Chore/improve pouchdb number of connections

### PR DESCRIPTION
**Pull request by guilbep**
# Describe your changes

Improve performances

# Related GitHub issues and pull requests

- Ref: #278

### CODE REVIEW

File: frontend/src/store/GhgReferenceModule.ts [hunk 1]
```diff
@@ -54,65 +50,30 @@ const getters: GetterTree<GhgReferenceState, RootState> = {
...
+  getDoc: (
+    context: ActionContext<GhgReferenceModuleState, RootState>,
+    id: string
+  ): ReferenceItemInterface | undefined => {
+    const foundItem = ghg_reference.find((item) => item._id === id);
+    if (foundItem) {
+      context.commit("SET_ITEM", foundItem);
     } else {
-      throw new Error(MSG_DB_DOES_NOT_EXIST);
+      console.log(MSG_COULD_NOT_FIND_ITEM);
```
Comment (blueur) [inline frontend/src/store/GhgReferenceModule.ts:69]: log ?

File: frontend/src/store/GhgModule.ts [hunk 2]
```diff
@@ -155,9 +155,9 @@ function getGenericCountries(
 const actions: ActionTree<ProjectsState, RootState> = {
   syncDB: (context: ActionContext<ProjectsState, RootState>) => {
     context.commit("INIT_DB");
-    context.state.localCouch?.onChange(function () {
-      context.dispatch("getCountries");
-    });
+    // context.state.localCouch?.onChange(function () {
```
Comment (blueur) [inline frontend/src/store/GhgModule.ts:158]: comment ?

File: frontend/src/components/overview_data/OverviewProjects.vue [hunk 3]
```diff
@@ -548,7 +548,7 @@ export default class OverviewProjects extends Vue {
     this.syncDB();
     this.retrieveData();
 
-    this.changes = this.db?.onChange(this.retrieveData);
+    // this.changes = this.db?.onChange(this.retrieveData);
```
Comment (blueur) [inline frontend/src/components/overview_data/OverviewProjects.vue:551]: comment ?

File: frontend/src/components/overview_data/OverviewProjects.vue [hunk 4]
```diff
@@ -526,9 +526,9 @@ export default class OverviewProjects extends Vue {
 
     // TODO: correct the getScorecards function to avoid this behavior
     this.getScorecards(id);
-    this.changes = this.db?.onChange(() => {
-      this.getScorecards(id);
-    });
+    // this.changes = this.db?.onChange(() => {
```
Comment (blueur) [inline frontend/src/components/overview_data/OverviewProjects.vue:529]: comment ?

**Review by blueur — APPROVED** [review]

File: frontend/src/store/GhgReferenceModule.ts (hunk 1 above)
Comment (guilbep) [reply frontend/src/store/GhgReferenceModule.ts:69]: I removed it

**Pull request merged**